### PR TITLE
Resource path in request body

### PR DESCRIPTION
**Pull request by HeikoTheissen**
Needs https://github.com/oasis-tcs/odata-abnf/pull/119

### CODE REVIEW

File: odata-url-conventions/4 Resource Path.md [hunk 1]
```diff
@@ -1030,16 +1031,20 @@ $filter=LastName%20eq%20'P%26G'&$select=FirstName,LastName
 ```
 :::
 
+If the resource path is passed in the request body, it MUST be percent-encoded,
+come before any query options and be separated from them by a `?`.
```
Comment (ralfhandl) [inline odata-url-conventions/4 Resource Path.md:1035]:
```suggestion
come before any query options, and be separated from them by a `?`.
```

Comment (ralfhandl) [reply odata-url-conventions/4 Resource Path.md:1035]: In other words: standard representation of a (relative) URL.

Can the resource path be a relative URL, and is it then resolved relative to the request URL?

Is only `<service-root>/$query` allowed, or can I use any URL suffixed with `/$query`?

**Review by ralfhandl — COMMENTED** [review]
See comments

**Review by ralfhandl — DISMISSED** [review]

**Comment by HeikoTheissen** [comment]
As agreed in TC 2024-10-16.

**Pull request closed without merging**